ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activitySection.component.ts

```diff
@@ -71,6 +75,9 @@ export class ActivitySectionComponent implements OnInit, OnDestroy {
                         block.shown = element.shown;
                         blockVisibilityChanged = true;
                     }
+                    if (block.enabled !== element.enabled) {
+                        block.enabled = element.enabled;
```

Comment:
should we rerender components as well as we do when "shown" is changed?

Reply:
Correct! Thank you, I'll fix this part.